definitions/npm/deep-freeze-strict_v1.x.x/flow_v0.59.x-/test_deep-freeze-strict_v1.x.x.js

```diff
@@ -0,0 +1,17 @@
+// @flow
+import freeze from 'deep-freeze-strict'
+
+const a = {a: 1}
+
+const frozen = freeze(a)
+
+const b: number = frozen.a
+
+// $ExpectError
+const c: string = frozen.a
+
+// $ExpectError
+frozen.b
+
+// $ExpectError
+frozen.a = 2
```

Comment:
I'd add a test checking that it's actually read-only.

Reply:
I must be missing something really obvious – isn't that what lines 16-17 are doing?

Reply:
Oh unless you mean that I should test that the frozen object is read-only, rather than just the type of the frozen object? How can I do that? I thought these test files were only testing static types, rather than the runtime logic. Do I have access to jest in this file?

Reply:
Hey @AndrewSouthpaw no hurry but just wanted to make clear that I'd love to get this merged and whatever change you need, I'm very happy to make. Genuinely not sure how to proceed currently though!

Reply:
I'm gonna go ahead and approve this, we can add other tests in another PR, for good measure.